docs/docs/kb/09-docker.md

```diff
@@ -31,18 +31,28 @@ docker run -ti ignitehq/cli chain serve
 
 When Docker is installed, you can build a blockchain with a single command.
 
+Ignite CLI, and the chains you serve with Ignite CLI, persist some files.
+When using the CLI binary directly, those files are located in `$HOME/.ignite`
+and `$HOME/.cache`, but in the context of docker it's better to use an other
+directory than `$HOME`, so we use `$HOME/sdh`. This folder should be created
+manually prior to the docker commands below, or else docker creates it with the
+root user.
+
+```bash
+mkdir $HOME/sdh
+```
+
 To scaffold a blockchain `planet` in the `/apps` directory in the container, run this command in a terminal window:
 
 ```bash
-docker run -ti -w /app -v $HOME/sdh:/home/tendermint -v $PWD:/app ignitehq/cli:0.16.0 app github.com/hello/planet
+docker run -ti -v $HOME/sdh:/home/tendermint -v $PWD:/apps ignitehq/cli:0.16.0 scaffold chain github.com/hello/planet
```

Comment:
Shouldn't it be `$PWD:/app` instead of `$PWD:/apps`? Otherwise the [line 56](https://github.com/ignite/cli/pull/2622/files#diff-ca6d8a18d0de8bcbaf90bcfc05dadfe29cb351917c9b955ded6ca8a40aa0073eR56) should be fixed

Reply:
The `Dockerfile` specifies `/apps` as the default working dir.

https://github.com/ignite/cli/blob/b4567b4e2cd9a39d790f104c62550aceda3ab858/Dockerfile#L41

There's no reason to change that working dir to `/app`, unless we prefer the name `app` over `apps` (which IMO makes more sense, afaik we don't deal with multiple apps in the context of Docker). But in that case we should rather change the Dockerfile and set the default working dir to `/app`.